### ISSUE TITLE
Should be able to focus() dropdown programmatically

### ISSUE THREAD

**Issue by nick211** [PolymerElements/paper-dropdown-menu#73]
E.g.

<paper-button>Set focus</paper-button>

<paper-dropdown-menu>
  <paper-menu class="dropdown-content">
    <paper-item>one</paper-item>
    <paper-item>two</paper-item>
  </paper-menu>
</paper-dropdown-menu>

<script>
  document.querySelector('paper-button').addEventListener('click', function() {
    var dropdown = document.querySelector('paper-dropdown-menu');
    dropdown.focus(); // Doesn't work
    // Workaround:
    //dropdown.$$('paper-input').inputElement.focus();  
  });
</script>

**Comment by SandeepSuddala** [comment]
+1

**Comment by cdata** [comment]
Good point. The dropdown should probably auto-forward-focus to its trigger.

**Comment by nerdkid93** [comment]
+1

**Comment by shqear93** [comment]
+1
